test/msw-api/setup-worker/start/quiet.fn.true.test.ts

```diff
@@ -0,0 +1,84 @@
+import * as path from 'path'
+import { TestAPI, runBrowserWith } from '../../../support/runBrowserWith'
+import { captureConsole } from '../../../support/captureConsole'
+
+let runtime: TestAPI
+
+beforeAll(async () => {
+  runtime = await runBrowserWith(
+    path.resolve(__dirname, 'quiet.fn.true.mocks.ts'),
+  )
+})
+
+afterAll(() => {
+  return runtime.cleanup()
+})
+
+test('log the captured request when the "quiet" option is returns to "false"', async () => {
+  const { messages } = captureConsole(runtime.page)
+
+  await runtime.page.evaluate(() => {
+    // @ts-ignore
+    return window.__MSW_REGISTRATION__
+  })
+
+  await runtime.reload()
+
+  const activationMessage = messages.startGroupCollapsed.find((text) => {
+    return text.includes('[MSW] Mocking enabled.')
+  })
+  expect(activationMessage).toBeFalsy()
+
+  const res = await runtime.request({
+    url: `${runtime.origin}/user`,
+  })
+
+  const headers = res.headers()
+  const body = await res.json()
+
+  expect(headers).toHaveProperty('x-powered-by', 'msw')
+  expect(body).toEqual({
+    firstName: 'John',
+    age: 32,
+  })
+
+  const requetsLog = messages.startGroupCollapsed.find((text) => {
+    return text.includes('[MSW]') && text.includes('GET /user')
+  })
+  expect(requetsLog).toBeTruthy()
+})
+
+test('does not log the captured request when the "quiet" option is returns to "true"', async () => {
+  const { messages } = captureConsole(runtime.page)
+
+  await runtime.page.evaluate(() => {
+    // @ts-ignore
+    return window.__MSW_REGISTRATION__
+  })
+
+  await runtime.reload()
+
+  const activationMessage = messages.startGroupCollapsed.find((text) => {
+    return text.includes('[MSW] Mocking enabled.')
+  })
+  expect(activationMessage).toBeFalsy()
+
+  const res = await runtime.request({
+    url: `${runtime.origin}/foo`,
+  })
+
+  const headers = res.headers()
+  const body = await res.json()
+
+  expect(headers).toHaveProperty('x-powered-by', 'msw')
+  expect(body).toEqual({
+    firstName: 'John',
+    age: 32,
+  })
+
+  const requetsLog = messages.startGroupCollapsed.find((text) => {
+    return text.includes('[MSW]') && text.includes('GET /foo')
+  })
+
+  expect(requetsLog).toBeUndefined()
+})
```

Comment:
yeah this looks great, will update. What does `await runtime.reload()` do? Do I have to remove it?

Reply:
it will reload the page, because we have a single instance of puppeteer it is necessary to have everything clean between test. It should be called before the `evaluate`. 

So when the test runs it will reload the page and then start MSW.

Reply:
I think that we can add in a `beforeEach` instead
```js
beforeEach(() => {
  await runtime.reload()
})
```

Reply:
We can also introduce a different test suite like `quiet-predicate` to scope the predicate functionality.

Reply:
both works for me, I created new file because I thought that was convention that being followed.
eg. `wait-until-ready.false.mocks.ts` and `wait-until-ready.error.mocks.ts`
let me know which works better?